packages/@o3r/third-party/src/bridge/iframe/bridge.ts

```diff
@@ -75,7 +75,7 @@ export class IframeBridge {
         return;
       } catch {}
     }
-    return Promise.reject('Handshake failed.');
```

Comment:
If I'm not mistaken we said that we should not change code like that 
```js
new Promise(function(resolve, reject) {
  setTimeout(function() {
    reject('reason');
  }, 1000);
}).catch(function(e) {
  console.log(e);
});
```
into 
```js 
new Promise(function() {
  setTimeout(function() {
    throw 'reason';
  }, 1000);
}).catch(function(e) {
  console.log(e);
});
```
because in the 2nd case the `catch` will not be called 
but here we are not inside a `new Promise` 
![image](https://github.com/user-attachments/assets/b630b684-2e3e-4b37-8c17-8937c3874582)

Reply:
I was referring to https://eslint.org/docs/latest/rules/prefer-promise-reject-errors that we choose to disable.
I understand that here the only error caught by the linter was the lack of `new Error`
You decided to change the `return Promise.reject` to `throw` on top of it but not due to an eslint error, right ?

Reply:
In fact the rule you mentioned is disable but we have the one for typescript [@typescript-eslint/prefer-promise-reject-errors](https://typescript-eslint.io/rules/prefer-promise-reject-errors) activated 
that's why I did the change from string to Error 
I decided to change to throw on my own it was not raised by the linter indeed

Reply:
sorry I meant https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-useless-promise-resolve-reject.md, clipboard issues :)
but I got it now, thanks for the explanation